common/test/acceptance/tests/studio/test_studio_settings.py

```diff
@@ -290,3 +290,10 @@ def set_wrong_inputs_to_fields(self):
                 "Course Announcement Date": '"string"',
             }
         )
+
+    def test_only_expected_fields_are_displayed(self):
+        expected_fields = self.advanced_settings.get_expected_settings_names()
```

Comment:
I've added the docstring and additional assertion -- thanks for pointing them out!